### PR TITLE
docs: add computer use guide

### PR DESCRIPTION
**Pull request by nicoalbanese**

### CODE REVIEW

File: content/docs/02-guides/05-computer-use.mdx [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        return {
+          type: 'image',
+          data: getScreenshot(),
+        };
+      }
+      default: {
+        return executeComputerAction(action, coordinate, text);
+      }
+    }
+  },
```
Comment (lgrammel) [inline content/docs/02-guides/05-computer-use.mdx:109]: this needs the experimental convert or it won't work

**Review by lgrammel — APPROVED** [review]

File: content/docs/02-guides/05-computer-use.mdx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+For streaming responses, use `streamText` to receive updates in real-time:
+
+```ts
+const result = await streamText({
+  model: anthropic('claude-3-5-sonnet-20241022'),
+  prompt: 'Open the browser and navigate to vercel.com',
+  tools: { computer: computerTool },
+});
+
+for await (const chunk of stream) {
```
Comment (shaper) [inline content/docs/02-guides/05-computer-use.mdx:153]: nit: `stream` not defined above, maybe `const { stream } = await ...`?

Comment (nicoalbanese) [reply content/docs/02-guides/05-computer-use.mdx:153]: this is a great spot! I had renamed this later as I was writing. Will fix.

Comment (nicoalbanese) [reply content/docs/02-guides/05-computer-use.mdx:153]: https://github.com/vercel/ai/pull/3410

**Pull request merged**